src/pages/resources/upgrade-platform-sdks/v4-faq.md

```diff
@@ -14,7 +14,7 @@ To upgrade to the Experience Platform SDKs, see [Upgrade to the Experience Platf
 
 ## Will Mobile Services functionality still work?
 
-Yes! This end-of-support statement does not affect your use of Mobile Services in any way. You may continue to use features such as marketing links, push notifications, and in-app messaging by leveraging the Adobe Analytics - Mobile Services extension for the Experience Platform Mobile SDKs. Note that you will need to migrate from version 4 SDKs to Adobe Experience Platform SDKs after August 2021.
+No! Adobe Mobile Service's end-of-life date is [December 31, 2022](https://experienceleague.adobe.com/docs/mobile-services/using/eol.html).
```

Comment:
SUPER minor nit, but I feel like having an exclamation mark for "No!" is a bit too... exciting? I'd go with "No. Adobe Mobile Service's..." since it makes it a bit better.